### DISCUSSION TITLE
Java 任务执行

### DISCUSSION THREAD

**Discussion by xywhnh** [apache/dolphinscheduler#10518] [Q&A]
大佬们好！我们之前用的是xxl-job ,由于任务量比较大，XXL-JOB基于数据库悲观锁方式的性能有点达不到，想换成无中心化的定时任务
由于基于xxl-job 写了大量的定时任务执行器（基于java语言的 xxl-job JobHandler）。
大概看了一下[dolphinscheduler] 的任务类型，python shell 的都有，唯独没有看到Java 类型的。想问一下各位大佬，这种基于java 实现的执行器应该 用dolphinscheduler 的哪种任务类型呢？

**Comment by Tandoy** [comment]
see if this is the same requirement
https://github.com/apache/dolphinscheduler/discussions/10769
